esi_leap/objects/contract.py

```diff
@@ -62,6 +67,20 @@ def get_all_by_status(cls, context, status):
             context, status)
         return cls._from_db_object_list(context, db_contracts)
 
+    @classmethod
+    def retrieve_from_flocx_market(cls):
```

Comment:
Should this method belong to common/flocx-market? Since it's not relevant to contract in provider's database.

Reply:
Yep, I think so!

Reply:
I've moved this method to common/flocx_market.py. I will add tests for common/flocx_market right away.